tests/test_cli/test_main.py

```
@@ -0,0 +1,200 @@
+    assert "Error:" in result.output
+
+def test_cli_invalid_quarters_range(cli_runner, config_file):
+    """Test CLI with quarters out of range."""
+    result = cli_runner.invoke(main, [
+        "--config-file", config_file,
+        "--quarters", "0,5"
+    ])
+    assert result.exit_code != 0
+    assert "Error:" in result.output
+
+def test_cli_directory_creation(cli_runner, config_file, tmp_path, mocker):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_cli_directory_creation`

Add return type annotation: `None`

(ANN201)